kolibri_explore_plugin/assets/src/views/ContentLightbox.vue

```diff
@@ -94,6 +87,8 @@
     width: 85%;
     max-height: calc(100vh - 2rem);
     margin: 0 auto;
+    color: $white;
```

Comment:
There's a problem with this style, it's applied to the lightbox content, for example in some `practice` the question is not visible because it's white text over white background.

![imagen](https://user-images.githubusercontent.com/50350/121868316-212a1700-cd01-11eb-9b7e-0df003a3c516.png)

Reply:
Good catch! I've been trying only one exercise and I thought that area was just empty. I will check with other examples.